docs/DeveloperGuide.md

```diff
@@ -312,11 +312,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-*{More to be added}*
+1. Should work if Java `11` or above is installed.
+1. Should be platform-independent, minimally working on any _mainstream OS_.
```

Comment:
Perhaps the numbering here should be in numerical order? 😄

Reply:
Noted. I'll change it.